tests/mgbench/benchmark.py

```diff
@@ -243,9 +259,9 @@ def filter_benchmarks(generators, patterns):
 
     # Run all benchmarks in all available groups.
 
-    for with_fine_grained_authorization in [False, True]:
+    for with_fine_grained_authorization in [False]:
```

Comment:
This is very strange (from before), that hole block could/should be massively simplified

Reply:
Yes, I have an idea how to make it a bit different, I will change tomorrow/or next week.